src/app/components/VisuallyHiddenText/index.jsx

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
```

Comment:
This component should also have a test file as well as a stories file. 

The reason for the stories file despite the text not being visible within storybook, is that the existence of the component would be made clear and also to enable easier accessibility testing of this component in isolation.